### DISCUSSION TITLE
mishards periodic slow query

### DISCUSSION THREAD

**Discussion by zhangchenglala** [milvus-io/milvus#4143] [Q&A and General discussion]
There is a phenomenon in the mishard running test. Every 5 minutes, the request will usher in a larger wave of delay, and a single request exceeds 1000ms.
The query log found that milvus has fewer slow queries.

mishard运行测试中有一个现象，每隔5min，请求就会迎来一波较大的时延，单次请求超过1000ms。
查询日志发现，milvus慢查询较少。

> [root@10.16.135.92~]# cat mishards_running.log | grep "SearchVector takes" | awk '{print $7}' | sort | tail -n 10
0.25246667861938477
0.2600548267364502
0.2609741687774658
0.26666879653930664
0.2794218063354492
0.3043513298034668
0.45690035820007324
0.4811515808105469
5.0672712326049805
5.08876371383667

milvus 0.10.3
mi:1，rw:1，ro:2

**Comment by yhmo** [comment]
Is there any other actions along with search? For example, one client continually insert vectors, the other continually search.
How many vectors in the collection? Same search parameter every time?
